packages/zowe-explorer/CHANGELOG.md

```diff
@@ -61,6 +64,7 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 - Fixed issue where multiple extensions that contribute profiles to a tree view using the Zowe Explorer API may fail to load. [#2888](https://github.com/zowe/zowe-explorer-vscode/issues/2888)
 - Fixed regression where `getProviderForNode` returned the wrong tree provider after performing an action on a Zowe tree node, causing some commands to fail silently. [#2967](https://github.com/zowe/zowe-explorer-vscode/issues/2967)
 - Update Zowe SDKs to `8.0.0-next.202407051717` for technical currency. [#2918](https://github.com/zowe/zowe-explorer-vscode/issues/2918)
+- Update Zowe SDKs to `8.0.0-next.202407232256` for technical currency. [#2994](https://github.com/zowe/zowe-explorer-vscode/pull/2994)
```

Comment:
Do we want to keep the line above this one? It seems kind of redundant but I suppose it's good to keep a link to both of the relevant issues/PRs for updating SDKs 😋